CIP-????/README.md

```diff
@@ -0,0 +1,246 @@
+---
+CIP: ????
+Title: Encrypted Transaction message/comment metadata (Addendum to CIP-0020)
+Status: Proposed
```

Comment:
According to the new CIP-0001 this still needs a `Path to Active` - regardless of whether it's `Active` or `Proposed`.

Reply:
Ok, i will add a "Path to Active" section. But just wondering what the acceptance criteria whereby a proposal becomes 'Active' should look like? Currently we have cexplorer and the spo scripts which have it implemented. adastat is right now looking into it to add it, also cardanoscan/typhon-wallet. Ola is Co-Author of the CIP, so its also on the ToDo list of eternl wallet.

Reply:
in the meantime also adastat has is running on a dev server, active on mainnet soon 😄

Reply:
@rphair i have added the "Path to Active" section